pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/NamespacesBase.java

```diff
@@ -538,13 +545,13 @@ protected void internalDeleteNamespaceForcefully(AsyncResponse asyncResponse, bo
 
         FutureUtil.waitForAll(bundleFutures).thenCompose(__ -> internalClearZkSources()).handle((result, exception) -> {
             if (exception != null) {
-                if (exception.getCause() instanceof PulsarAdminException) {
-                    asyncResponse.resume(new RestException((PulsarAdminException) exception.getCause()));
-                    return null;
+                Throwable cause = FutureUtil.unwrapCompletionException(exception);
+                if (cause instanceof PulsarAdminException.ConflictException) {
+                    internalDeleteNamespaceForcefully(asyncResponse, authoritative);
```

Comment:
forget `return` here?

Reply:
And if it's a risk to use `metadata` thread do delete the namespace again?

Reply:
@mattisonchao fixed.